test/test_pub.dart

```diff
@@ -210,6 +210,20 @@ Future<PubProcess> pubRun({bool global = false, Iterable<String> args}) async {
   return pub;
 }
 
+/// Schedules starting the "pub run --v2" process and validates the
+/// expected startup output.
+///
+/// Returns the `pub run` process.
+Future<PubProcess> pubRunV2({Iterable<String> args}) async {
+  final pub = await startPub(args: ['run', '--v2', ...args]);
+
+  // Loading sources and transformers isn't normally printed, but the pub test
```

Comment:
transformers have been removed. Was this copied from somewhere? Is it still necessary?

Reply:
Blatantly stolen from `pubRun` above, but yes, it would be interesting to see if this is necessary.